llama/cli.py

```diff
@@ -1,6 +1,14 @@
 import click
 
+from llama import marc_export
+
 
 @click.group()
 def cli():
+    # move date retrieval here since it will be shared by multiple commands?
     pass
+
+
+@cli.command()
```

Comment:
I would rename this to something like `concat_timdex_export` and add a required option of `export_type` (a choice type, either `UPDATE` or `FULL`)